src/overrides.ts

```diff
@@ -19,14 +19,15 @@ import { SpeciesId } from "#enums/species-id";
 import { Stat } from "#enums/stat";
 import { StatusEffect } from "#enums/status-effect";
 import { TimeOfDay } from "#enums/time-of-day";
+import { TrainerType } from "#enums/trainer-type";
 import { Unlockables } from "#enums/unlockables";
 import { VariantTier } from "#enums/variant-tier";
 import { WeatherType } from "#enums/weather-type";
 
 /**
  * This comment block exists to prevent IDEs from automatically removing unused imports
  * {@linkcode BerryType}, {@linkcode ElementalType}, {@linkcode EvolutionItem}
- * {@linkcode FormChangeItem}, {@linkcode Stat}
+ * {@linkcode FormChangeItem}, {@linkcode Stat}, {@linkcode TrainerType}
```

Comment:
```suggestion
 * {@linkcode FormChangeItem}, {@linkcode Stat}
```
That shouldn't be necessary since you're using it as the type of `TRAINER_TYPE_OVERRIDE`. It's only needed for the ones that aren't used (there's no override that is typed as `ElementalType` for example).

Reply:
Line 274, yeah